dbt/adapters/contracts/relation.py

```diff
@@ -1,12 +1,15 @@
+from abc import ABC
```

Comment:
Minor nit, these imports are out of order. `dbt_common.contracts.config.materialization` is 3rd party, should be moved down before `dbt_common.contracts.util`. The extra line on line 2 is unnecessary. The imports for `typing` are not alphabetical.